test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/vars/vars_to_delete.yml

```diff
@@ -0,0 +1,3 @@
+---
+instance_id: i-0fe387534190cc512
```

Comment:
I think we are not supposed to push to this file into Git. Please remove this file